packages/cli/src/bulk.ts

```diff
@@ -33,29 +33,52 @@ bulk
 bulk
   .command('import')
   .argument('<filename>', 'File Name')
-  .action(async (fileName) => {
+  .option(
+    '--numResourcesPerRequest <numResourcesPerRequest>',
+    'optional number of resources to import per batch request. Defaults to 25.'
```

Comment:
I believe the 3rd argument of option can is the default value and you can set '25' after the description. More on that here 

https://github.com/tj/commander.js/#other-option-types-negatable-boolean-and-booleanvalue

Reply:
This is great! It simplifies the logic to check for `undefined`